sympy/functions/special/error_functions.py

```diff
@@ -84,7 +84,7 @@ class erf(Function):
 
     >>> from sympy import diff
     >>> diff(erf(z), z)
-    2*exp(-z**2)/sqrt(pi)
+    2/(sqrt(pi)*exp(z**2))
```

Comment:
This printing seems odd to me

Reply:
Is this just for the str printer and not the latex and pretty printers? It might be confusing for them to be different.

Reply:
Does it look odd because of the `exp`? How about
```python
>>> 2*3**(-z)/sqrt(pi)
2/3**z
>>> x*y**(-3)
x/y**3
```

Reply:
> Is this just for the str printer

yes, until the changes are acceptable

Reply:
Yes, I would usually write an exponential function like `e^{-x}` to be in the numerator even if the argument is negated.

Reply:
> usually write an exponential function like `e^{-x}` to be in the numerator

what about the other two examples involving powers?

Reply:
I think that the other two look good. Actually the latex and pretty printers already show `y**-3` in the numerator.